scripts/run_backend_tests.py

```diff
@@ -659,6 +695,25 @@ def check_coverage(
     process = subprocess.run(
         cmd, capture_output=True, encoding='utf-8', env=env,
         check=False)
+
+    not_covered_lines = []
```

Comment:
I don't understand what this is storing, perhaps add a comment and improve the naming.

Reply:
@seanlip i tried to explain this in updated code.